draft-ietf-rtcweb-jsep.xml

```diff
@@ -395,6 +395,12 @@ setRemote(ANSWER) |                                   |
           </figure>
         </t>
 
+        <t> In addition to theses state transitions, there is a
+        rollback operation, described in <xref
+        target="sec.rollback"/>, that returns the system to the stable
+        state. Rollback is typically used to recover from errors
+        produced by the remote side.</t>
```

Comment:
This isn't always true though. Maybe, 'typically used to recover when an offer is rejected'?